ipatests/test_integration/test_membermanager.py

```diff
@@ -212,3 +212,51 @@ def test_hostgroup_member_manager_nopermission(self):
             "'write' privilege to the 'memberManager' attribute of entry"
         )
         assert expected in result.stdout_text
+
+    def test_member_manager_upgrade_scenario(self):
```

Comment:
please write doc string specifying what you are testing here.

Reply:
Sure, TBD before review.